keywords/ClusterKeywords.py

```diff
@@ -47,24 +49,41 @@ def unset_cluster_config(self):
         log_info("Unsetting CLUSTER_CONFIG")
         del os.environ["CLUSTER_CONFIG"]
 
-    def get_cluster_topology(self, cluster_config):
+    def get_cluster_topology(self, cluster_config, lb_enable=True):
         """
         Returns a dictionary of cluster endpoints that will be consumable
           ${sg1} = cluster["sync_gateways"][0]["public"]
           ${sg1_admin} = cluster["sync_gateways"][0]["admin"]
           ${ac1} = cluster["sg_accels"][0]
           ${cbs} = cluster["couchbase_servers"][0]
+
+          Setting lb_enable to True will return SG IPs instead of lb IPs
+          install_nginx sets it to False to get the SG_IPs for the nginx.conf
         """
 
         with open("{}.json".format(cluster_config)) as f:
             cluster = json.loads(f.read())
 
         sg_urls = []
 
-        for sg in cluster["sync_gateways"]:
-            public = "http://{}:4984".format(sg["ip"])
-            admin = "http://{}:4985".format(sg["ip"])
-            sg_urls.append({"public": public, "admin": admin})
+        # Get load balancer IP
+        lb_ip = None
+        if is_load_balancer_enabled(cluster_config) and lb_enable:
```

Comment:
lb_enable can be used to override the behavior of adding lb IPs even if load balancer is enabled. install_nginx sets it to False to get the SG_IPs for the nginx.conf.